web/admin.php

```diff
@@ -75,7 +74,7 @@
     if ($Request->query->has('q')) {
         $isSearching = true;
         $usersArr = $App->Users->readFromQuery(
-            filter_var($Request->query->get('q'), FILTER_SANITIZE_STRING),
+            (string) $Request->query->get('q', ''),
```

Comment:
you can use `query->getString` here methinks

Reply:
That will not work with email address search and names that contain any punctuation like "O'Brian" or "Smith-Brown".

Reply:
What do you mean? It works fine, and in any case getString is simply letting symfony doing the cast to string, so code is exactly the same.

![2024-01-02-090653_858x349_scrot](https://github.com/elabftw/elabftw/assets/3043706/84e3fc52-c206-49e3-b4dd-0755ae21dd3a)

Reply:
Yes, of course. I was thinking about getAlpha() or getAlnum().